src/templates/app/cashier/index.jsx

```diff
@@ -139,6 +139,30 @@ const Cashier = () => (
                 <DepositWithdrawButton is_payment_agent />
             </div>
         </div>
+
+        <div className='gr-padding-10' />
+
+        <div className='gr-padding-10 table-body invisible gr-parent' id='dp2p_info'>
+            <h3 className='gr-padding-10'>{it.L('DP2P')}</h3>
+            <div className='gr-row'>
+                <IconWithLink img_id='virtual_money_icon' img_src={it.url_for('images/pages/cashier/dp2p_logo.svg')} />
```

Comment:
why are we repeating the img_id?

Reply:
I mean, why is it called virtual_money_icon

Reply:
my mistake, should not need img_id for this